examples/models/turbine.py

```diff
@@ -301,194 +302,11 @@ def dwr_indicator(mesh, q, q_star):
     """
     mesh_plus = q.function_space().mesh()
 
-    # Extract parameters from solver object
+    # Extract indicator in enriched space
```

Comment:
The function no longer depends on the `turbine` test case. It could be used for other test cases, too.